src/Converter.php

```diff
@@ -29,6 +29,25 @@ public function getAsText(): string
         return $output;
     }
 
+    public function getSubstringHtml(string $numberOfCharacters, bool $escape = false): string
```

Comment:
Why `$numberOfCharacters` is a string?

Reply:
Because i'm an idiot. I'll change to a int